src/one_dragon/base/operation/one_dragon_context.py

```
@@ -505,6 +515,79 @@ def init_ocr(self) -> None:
+                # 当前用 V5 顶住 后台下载 V6 成功后自动切换
+                self._download_ocr_v6_in_background()
+
+    def _decide_ocr_model_name(self) -> str:
+        """
+        决定本次初始化使用的 OCR 模型名
+
+        调试模式: 直接使用配置里的模型 不自动切换
+        正常模式: 向 V6 收敛
+          - V6 文件齐全 -> 用 V6
+          - V6 不齐 但 V5 齐全 -> 先用 V5 顶住 后台下载 V6
+          - 都没有 -> 直接用 V6（会触发下载）
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)